port/unix/omrfile.c

```diff
@@ -353,7 +356,7 @@ omrfile_open(struct OMRPortLibrary *portLibrary, const char *path, int32_t flags
 #endif
 
 	if (-1 == fd) {
-		Trc_PRT_file_open_Exception2(path, errno, findError(errno));
```

Comment:
Did you mean to remove the function call? This is common code and the change will have an impact on other platforms will it not?